lib/bridge/MatrixHandler.js

```diff
@@ -369,7 +368,90 @@ MatrixHandler.prototype._onAdminMessage = Promise.coroutine(function*(req, event
         return;
     }
     else {
-        req.log.info("No valid admin command: %s", event.content.body);
+        let fmt = '\nirc.server\n'+
+                    '(COMMAND [arg0 [arg1 [...]]])\n'+
+                    '(COMMAND [arg0 [arg1 [...]]])\n'+
+                    'irc.server2';
```

Comment:
You may find using ` easier here since then you can put actual new lines rather than escaping them:

```javascript
> s = `foo
... bar
... tar`
'foo\nbar\ntar'
```